src/OnboardingSPA/components/SocialMediaForm/urlValidator.js

```diff
@@ -0,0 +1,158 @@
+const SocialMediaSites = {
+	FACEBOOK: 'facebook',
```

Comment:
can this file be moved to the utils folder?

Reply:
I assume that as this is a very specific helper function for the component SocialMediaForm itself and won't be reused anywhere else it was a more apt location for the file.